programs/virtual-curve/src/instructions/partner/ix_create_config.rs

```diff
@@ -31,11 +32,73 @@ pub struct ConfigParameters {
     pub creator_locked_lp_percentage: u8,
     pub migration_quote_threshold: u64,
     pub sqrt_start_price: u128,
+    pub locked_vesting: LockedVestingParams,
     /// padding for future use
-    pub padding: [u64; 6],
+    pub padding: u64,
     pub curve: Vec<LiquidityDistributionParameters>,
 }
 
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, Copy, Debug, Default, PartialEq)]
+pub struct LockedVestingParams {
+    pub amount_per_period: u64,
+    pub cliff_duration_from_migration_time: u64,
+    pub frequency: u64,
+    pub number_of_period: u64,
+    pub cliff_unlock_amount: u64,
+}
+
+impl LockedVestingParams {
+    pub fn to_locked_vesting_config(&self) -> LockedVestingConfig {
+        LockedVestingConfig {
+            amount_per_period: self.amount_per_period,
+            cliff_duration_from_migration_time: self.cliff_duration_from_migration_time,
+            frequency: self.frequency,
+            number_of_period: self.number_of_period,
+            cliff_unlock_amount: self.cliff_unlock_amount,
+            ..Default::default()
+        }
+    }
+
+    pub fn to_create_vesting_escrow_params(
+        &self,
+        finish_curve_timestamp: u64,
+    ) -> Result<CreateVestingEscrowParameters> {
+        let cliff_time =
+            finish_curve_timestamp.safe_add(self.cliff_duration_from_migration_time)?;
+        Ok(CreateVestingEscrowParameters {
+            vesting_start_time: finish_curve_timestamp,
+            cliff_time,
+            frequency: self.frequency,
+            cliff_unlock_amount: self.cliff_unlock_amount,
+            amount_per_period: self.amount_per_period,
+            number_of_period: self.number_of_period,
+            update_recipient_mode: 1, // only creator
```

Comment:
This means that only the `pool_authority` can update. Is it intended to make the vesting escrow immutable?

Reply:
yes, we make as the same as catstarter (in case request change in future)